packages/babel-preset-mc-app/index.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable global-require */
-module.exports = function getBabePresetConfigForMcApp() {
+module.exports = function getBabePresetConfigForMcApp(api, opts) {
```

Comment:
We don't use `env`. The `opts` is what comes from e.g. webpack. Maybe we can use `env` in the future.